packages/visualizations/tests/dummy/app/templates/table.hbs

```diff
@@ -1,4 +1,14 @@
 {{!-- Copyright 2017, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+<div class='table-config__totals-toggle-section'>
```

Comment:
do we have any tests for this feature?

Reply:
Will add them once other issues be addressed